docker-compose.yml

```diff
@@ -2,14 +2,13 @@ version: "3.8"
 services:
   mongo:
     image: mongo:4.4
-    container_name: mongo
+    container_name: mongo_l
```

Comment:
?

Reply:
just to show that this container created for logistic servcie